### ISSUE TITLE
Deprecation warning upon installation

### ISSUE THREAD

**Issue by mudrd8mz** [bizDevelop/moodle-mod_bizexaminer#4]
During the installation, the following error is reported:

```
Deprecated: explode(): Passing null to parameter #2 ($string) of type string is deprecated in mod/bizexaminer/classes/admin/admin_setting_api_credentials.php on line 138
```

Please make sure to have the plugin tested with the full developer debugging enabled and displayed.

Ref: https://moodledev.io/general/community/plugincontribution/checklist#functionality

**Comment by mudrd8mz** [comment]
Also appears when using the module:

![image](https://github.com/bizDevelop/moodle-mod_bizexaminer/assets/85831/e88bed35-7e7a-4537-8ae6-19acca6ab4ce)

**Comment by gaambo** [comment]
Fixed in [v1.3.0](https://github.com/bizDevelop/moodle-mod_bizexaminer/releases/tag/v1.3.0)
